src/Factory/ZddMessageFactory.php

```diff
@@ -26,9 +28,12 @@ public function create(string $className): ZddMessage
     {
         $propertyList = $this->propertyExtractor->extractPropertiesFromClass($className);
 
-        $message = (new \ReflectionClass($className))->newInstanceWithoutConstructor();
-        foreach ($propertyList->getProperties() as $property) {
-            $this->forcePropertyValue($message, $property->name, $property->value);
+        $message = $this->config->generateCustomMessage($className);
+        if (null === $message) {
```

Comment:
You should precise in phpdoc of `ZddMessageConfigInterface::generateCustomMessage` that the method should return `null` when they want to use the reflection otherwise it will not work.